crates/dslab-core/src/state.rs

```diff
@@ -74,9 +76,53 @@ impl SimulationState {
         }
     }
 
+    pub fn add_ordered_event<T>(&mut self, data: T, src: Id, dest: Id, delay: f64) -> EventId
+    where
+        T: EventData,
+    {
+        if !self.can_add_ordered_event(delay) {
+            panic!("Event order is broken! Pass events to emit_*_ordered functions in non-decreasing order of time.");
```

Comment:
Предлагаю не ссылаться здесь на emit_ordered_*, все таки это не часть интерфейса самого SimulationState, Думаю из сообщения и так будет понятно в чем дело.

```suggestion
            panic!("Event order is broken! Ordered events should be added in non-decreasing order of their time.");
```

Reply:
Поправил